bin/verify-exercises

```diff
@@ -1,45 +1,60 @@
 #!/usr/bin/env bash
 
-temp_dir_base=$(mktemp -d)
+set -euo pipefail
 
-run_test() {
-    slug=$(basename $1)
-    temp_dir=${temp_dir_base}/${slug}
-    mkdir -p ${temp_dir}
-    
-    cp -r "$1/." $temp_dir
+original_dir=$(pwd)
 
-    outpath=$temp_dir/src/$slug.lfe
-    if [ "$slug" = "binary" ]; then
-        outpath=$temp_dir/src/binary-string.lfe
+get_deprecated_exercises() {
+    if [ -f "config.json" ]; then
+        jq -r '.exercises.practice[] | select(.status == "deprecated") | .slug' config.json 2>/dev/null || true
     fi
+}
 
-    cp $temp_dir/.meta/example.lfe $outpath
-
-    (cd /opt/test-runner && bin/run.sh $slug $temp_dir $temp_dir) || exit 1
+run_test() {
+    exercise_dir="${1%/}"
+    pushd "$exercise_dir"
 
-    test_status="$(jq -r '.status' $temp_dir/results.json)"
+    slug=$(basename $1)
+    example_path=".meta/example.lfe"
+    stub_path="src/$slug.lfe"
+    stub_backup="src/$slug.bak"
+    if [ "$slug" = "binary" ]; then
+        stub_path="src/binary-string.lfe"
+        stub_backup="src/binary-string.bak"
```

Comment:
Temp files (`mktemp`) is the gold standard for temp files. It supports temp dirs, too.